pkg/sql/codegen/xgboost/codegen_explain.go

```diff
@@ -29,9 +28,6 @@ const (
 
 // Explain generates a Python program to explain a trained model.
 func Explain(explainStmt *ir.ExplainStmt, session *pb.Session) (string, error) {
-	if explainStmt.Explainer != "TreeExplainer" {
```

Comment:
`explainStmt.Explainer` is actually not used now, maybe add an issue about use the actual value.